metron-deployment/packaging/docker/rpm-docker/build.sh

```diff
@@ -36,7 +36,7 @@ if [ $? -ne 0 ] && [ $OWNER_UID -ne 0 ]; then
 fi
 
 rm -rf SRPMS/ RPMS/ && \
-rpmbuild -v -ba --define "_topdir $(pwd)" --define "_version ${VERSION}" --define "_prerelease ${PRERELEASE}" SPECS/metron.spec && \
+QA_SKIP_BUILD_ROOT=1 rpmbuild -v -ba --define "_topdir $(pwd)" --define "_version ${VERSION}" --define "_prerelease ${PRERELEASE}" SPECS/metron.spec && \
```

Comment:
This was another annoying issue.  When NPM builds the dependencies, it adds a `package.json` to each dependency.  In this `package.json`, there is a "_where" field that ends up containing the complete build path from where the RPMs are built.  

Based on some research I can find no way to get rid of this absolute path.  And I also know that Node/NPM does not actually use this for anything.  So it is harmless, but triggers the RPM checks, which I had to disable.

https://github.com/npm/npm/issues/10393

Is there a better way?

Reply:
Just saw this by chance. I can recommend using https://www.npmjs.com/package/removeNPMAbsolutePaths which is already use by other Apache projects like Cordova to remove the absolute paths created by NPM.